src/main/java/uk/gov/hmcts/reform/prl/utils/CaseUtils.java

```diff
@@ -796,4 +796,9 @@ public static List<String> mapAmUserRolesToIdamRoles(RoleAssignmentServiceRespon
         return roles;
     }
 
+    public static boolean hasDashboardAccess(Element<PartyDetails> party) {
```

Comment:
check if there is an existing method to determine dashboard access in cos code

Reply:
Hi @yogendraU there is a method called isAccessEnabled in the serviceofapplicationservice but this is a private method